shaders/line.vert

```diff
@@ -0,0 +1,12 @@
+#version 330 core
+
+// Input vertex data, different for all executions of this shader.
+layout(location = 0) in vec3 vertexPosition_modelspace;
+
+void main(){
+	
+	gl_Position.xyz = vertexPosition_modelspace;
+	gl_Position.w = 1.0;
```

Comment:
Think it's unnecessary to make a new one. You can just use Daniel's line shader and should be the same. 

main.cpp::main()
`Shader* programLine = new Shader( "shaders/vertex.glsl", "shaders/linefrag.frag");`